### ISSUE TITLE
Creates invalid SARIF file if container scan returns no results

### ISSUE THREAD

**Issue by pascalberger** [rm3l/container-scan-to-sarif#15]
When container scan doesn't find any vulnerabilities the resulting SARIF file has `result` set to `null`:

```
{
  "version": "2.1.0",
  "$schema": "https://raw.githubusercontent.com/oasis-tcs/sarif-spec/master/Documents/CommitteeSpecifications/2.1.0/sarif-schema-2.1.0.json",
  "runs": [
    {
      "tool": {
        "driver": {
          "name": "container-scan",
          "informationUri": "https://github.com/Azure/container-scan",
          "fullName": "Container Scan",
          "version": "0.1",
          "rules": []
        },
        "extensions": [
          {
            "name": "Trivy",
            "version": "latest"
          },
          {
            "name": "Dockle",
            "version": "latest"
          }
        ]
      },
      "results": null
    }
  ]
}
```

The `upload-sarif` action expects an array in this case and fails with an error:

```
instance.runs[0].results is not of a type(s) array
```

**Comment by rm3l** [comment]
Thanks for reporting this issue - I'll look into it when I get a chance.

**Comment by rm3l** [comment]
@pascalberger This issue should be fixed now.
I've published a new version for this tool ([0.7.3](https://github.com/rm3l/container-scan-to-sarif/releases/tag/0.7.3)) and the `container-scan-to-sarif-action` Action ([1.7.1](https://github.com/rm3l/container-scan-to-sarif-action/releases/tag/v1.7.1)).
Please do let me know if the issue still persists.
